package.json

```diff
@@ -9,15 +9,16 @@
     "registry": "node ./dist/registry.js",
     "compile": "npx tsc",
     "test": "jest --verbose --coverage --no-cache",
+    "benchERC20": "node benchmarking/benchERC20.js",
     "bench": "node benchmarking/bench.js",
     "date": "node benchmarking/generateDateInRange.js"
   },
   "type": "module",
   "license": "MIT",
   "dependencies": {
-    "@airswap/libraries": "4.0.5-beta.5",
+    "@airswap/libraries": "4.0.5-beta.7",
     "@airswap/swap-erc20": "4.0.2",
-    "@airswap/types": "4.0.3-beta.0",
+    "@airswap/types": "4.0.3-beta.1",
     "@airswap/utils": "4.0.5-beta.1",
```

Comment:
Please upgrade to latest like so:
```
    "@airswap/libraries": "^4.0.0",
    "@airswap/types": "^4.0.0",
    "@airswap/utils": "^4.0.0",
```
`@airswap/swap-erc20` is included in `@airswap/libraries` so should not need to be added separately.